common/lib/xmodule/xmodule/tests/test_capa_module.py

```diff
@@ -2530,6 +2531,27 @@ def test_indexing_text_input(self):
             }
         )
 
+    def test_indexing_non_latin_problem(self):
+        sample_text_input_problem_xml = textwrap.dedent("""
+            <problem>
+                <script type="text/python">FX1_VAL='Καλημέρα'</script>
+                <p>Δοκιμή με μεταβλητές με Ελληνικούς χαρακτήρες μέσα σε python: $FX1_VAL</p>
+            </problem>
+        """)
+        name = "Non latin Input"
+        descriptor = self._create_descriptor(sample_text_input_problem_xml, name=name)
+        capa_content = " FX1_VAL='Καλημέρα' Δοκιμή με μεταβλητές με Ελληνικούς χαρακτήρες μέσα σε python: $FX1_VAL "
+        self.assertEquals(
```

Comment:
Let's validate the smallest thing possible, in this case, we're only interested in `[content][capa_content]`, right?

Reply:
yes, you are right

Reply:
@amir-qayyum-khan Do you think you can make that change to the test?

Reply:
Done @gsong